### ISSUE TITLE
The example from http://idyll-lang.org/editor/ should be buildable locally

### ISSUE THREAD

**Issue by aldanor** [idyll-lang/idyll#238]
I've tried copying the example page from [online editor](http://idyll-lang.org/editor/) into a `foo.idl` and compiling it, but it first failed with

```js
Error: Component named DisplayVar could not be found.
...
```

which I guess was because of `DisplayVar` being renamed to `Display`, and after fixing that it failed again with

```js
events.js:136
      throw er; // Unhandled 'error' event
      ^

Error: Cannot find module './src/ParseError' from '.node/lib/node_modules/idyll/node_modules/katex/dist'
...
```

which sounds like something related to `katex`, but I'm not so sure.

**Comment by mathisonian** [comment]
@aldanor I wonder if the second error you are seeing is related to #239? Another katex issue

**Comment by hmuar** [comment]
@aldanor Did you ever get this resolved? I'm running into same error. Trying solution in #239 didn't work out for me.

**Comment by mathisonian** [comment]
@hmuar can you give me more info on your local setup to try to reproduce? did you create the local project using the generator?

**Comment by hmuar** [comment]
@mathisonian Sure thing, I just followed the Quickstart. Everything else seems to work except [Equation] / katex. I've tried downgrading node to v6.x, still same issue. 

Mac OS 10.12.6

```
 ❯ node --version                                                                                                                                                         
v8.4.0

 ❯ npm --version                                                                                                                                                          
5.5.1

❯ npm install -g idyll
❯ npm install -g yo generator-idyll
❯ yo idyll
```

My Error:

```
 ❯ npm start                                                                                                                                                              [09:00:04]

> logistic-regression@1.0.0 start /Users/hemu/Documents/dev-react/idyll/logistic-regression
> idyll index.idl --defaultComponents=components/default --css styles.css --theme github --watch

events.js:182
      throw er; // Unhandled 'error' event
      ^

Error: Cannot find module './src/ParseError' from '/Users/hemu/Documents/dev-react/idyll/logistic-regression/node_modules/katex/dist'
    at /Users/hemu/Documents/dev-react/idyll/logistic-regression/node_modules/browser-resolve/node_modules/resolve/lib/async.js:55:21
    at load (/Users/hemu/Documents/dev-react/idyll/logistic-regression/node_modules/browser-resolve/node_modules/resolve/lib/async.js:69:43)
    at onex (/Users/hemu/Documents/dev-react/idyll/logistic-regression/node_modules/browser-resolve/node_modules/resolve/lib/async.js:92:31)
    at /Users/hemu/Documents/dev-react/idyll/logistic-regression/node_modules/browser-resolve/node_modules/resolve/lib/async.js:22:47
    at FSReqWrap.oncomplete (fs.js:152:21)
```
------

 Here is as far as I got investigating so far... but I not that familiar with `make` so getting stuck.

- react-latex is using katex@0.8.3.
- katex@0.8.3 is still using make, as opposed to current version where they migrated to npm scripts
- The package.json's `"prepublish": "make NIS= dist"` step leaves me with a `/Users/hemu/Documents/dev-react/idyll/logistic-regression/node_modules/katex/dist/katex.js` file that starts out
```
(function(f){if(typeof exports==="object"&&typeof module!=="undefined"){module.exports=f()}else if(typeof define==="function"&&define.amd){define([],f)}else{var g;if(typeof window!=="undefined"){g=window}else if(typeof global!=="undefined"){g=global}else if(typeof self!=="undefined"){g=self}else{g=this}g.katex = f()}})(function(){var define,module,exports;return (function e(t,n,r){function s(o,u){if(!n[o]){if(!t[o]){var a=typeof require=="function"&&require;if(!u&&a)return a(o,!0);if(i)return i(o,!0);var f=new Error("Cannot find module '"+o+"'");throw f.code="MODULE_NOT_FOUND",f}var l=n[o]={exports:{}};t[o][0].call(l.exports,function(e){var n=t[o][1][e];return s(n?n:e)},l,l.exports,e,t,n,r)}return n[o].exports}var i=typeof require=="function"&&require;for(var o=0;o<r.length;o++)s(r[o]);return s})({1:[function(require,module,exports){
"use strict";

var _ParseError = require("./src/ParseError");

var _ParseError2 = _interopRequireDefault(_ParseError);

var _Settings = require("./src/Settings");

var _Settings2 = _interopRequireDefault(_Settings);
```

And then it errors because there is no `./src/ParseError` in the dist directory.

If I manually replace katex@0.8.3 with the latest, katex@0.9.0, in my node_modules folder, I don't get the error I'm guessing because the build system is different and just uses npm scripts, but the equation looks deformed on screen.

So I think there is something about the katex build w/make step that is off for me?

**Comment by hmuar** [comment]
Alright, seems like the equation looks malformed when i try to use latest katex because of outdated stylesheet. These were my final steps to get [Equation][/Equation] working without the above error:

1. force react-latex to use katex@0.9.0
2. fix the malformed equation rendering by updating the katex stylesheet used in the default equation component in `components/default/equation.js`:

```
if (typeof document !== 'undefined') {
  document.write(
    '<link href="//cdnjs.cloudflare.com/ajax/libs/KaTeX/0.9.0/katex.min.css" rel="stylesheet">',
  );
}
```

**Comment by mathisonian** [comment]
The katex issues should be resolved in `v2.1.4`, now released on npm
